phost/src/main.rs

```diff
@@ -64,7 +76,7 @@ fn deopaque_signedblock(opaque_block: OpaqueSignedBlock) -> phala_node_runtime::
 }
 
 async fn get_block_at(client: &subxt::Client<Runtime>, h: Option<u32>)
```

Comment:
```suggestion
async fn get_block_at(client: &subxt::Client<Runtime>, h: Option<u32>, with_events: bool)
```

Add a param to specify if it's going to fetch events